Google/resource-manager/project/locals.tf

```diff
@@ -5,13 +5,18 @@ locals {
   projects_components_specs  = lookup(local.projects_components, "specs", {})
   projects_components_common = lookup(local.projects_components, "common", {})
 
-  parent_folder = var.parent_folder == null ? lookup(local.projects_components_common, "folder_id", null) : var.parent_folder
-
   projects_specs = {
     for project, config in local.projects_components_specs :
         project => merge(local.projects_components_common, config)
   }
 
+  projects_parent = {
+    for project, config in local.projects_components_specs : project => {
+      parent_folder = var.parent_folder == null ? lookup(config, "org_id", null) == null ? lookup(config, "folder_id", null) == null ? lookup(local.projects_components_common, "folder_id", null) : config.folder_id : null : var.parent_folder
```

Comment:
I think I'm missing some understanding here of multiple, subsequent `?`s

Reply:
putting common values aside for a moment, could this not just be a single `parent` attribute, where we say if `var.folder` is set, use it, if not, if `var.org` is set use, if not, do the same from the provided MCCF?

Reply:
> I think I'm missing some understanding here of multiple, subsequent ?s  

This is for the nested conditions explained in [comment above](https://github.com/mesoform/Multi-Cloud-Platform-Foundations/pull/20#discussion_r998022242)
So for `parent_folder` this was: If `var.parent_folder` is set, that is the value, if it is not set, check if `org_id` is present in the `specs` configuration. If `org_id` is present `parent_folder` is `null` (this is to ensure that `common` values are overridden correctly). If it isn't present, check if `folder_id` is present in `specs`. If `folder_id` is present in `specs`, set to that, otherwise check if it set in `commons`. If it is not set in `commons`, `parent_folder` is `null`

> putting common values aside for a moment, could this not just be a single parent attribute, where we say if var.folder is set, use it, if not, if var.org is set use, if not, do the same from the provided MCCF?  

There can be a single `parent` attribute, had previously separated it to try and make the conditions more readable, as it would lead to a long chain of conditions:
```terraform
parent = var.parent_folder == null ? var.parent_org == null ? lookup(config, "folder_id", null) == null ? lookup(config, "org_id", null) == null ? lookup(local.projects_components_common, "folder_id", null) == null ? lookup(local.projects_components_common, "org_id", null): local.projects_components_common.folder_id : config.org_id : config.folder_id : var.parent_org : var.parent_folder 
```
So should I change to using the single `parent` attibute?

Reply:
I'd missed your other comment (sometimes doesn't show all comments well on the phone). Fine to leave as is but lets document the order of priority with an example of how to use (e.g. Our use case)

Reply:
Added documentation to readme